templates/src/python/grpcio_tests/grpc_version.py.template

```diff
@@ -0,0 +1,34 @@
+%YAML 1.2
+--- |
+  # Copyright 2015, Google Inc.
```

Comment:
This template itself is new-in-2016 content even if the license text is older, right?